docs/install/index.md

```diff
@@ -84,7 +84,7 @@ $ wget https://bootstrap.pypa.io/get-pip.py && sudo python get-pip.py
 **Step 2** Install MXNet with OpenBLAS acceleration.
 
 ```bash
-$ pip install mxnet
+$ sudo pip install mxnet
```

Comment:
sudo -H?

Reply:
@marcoabreu Comments about using `sudo -H`

Reply:
I do not have an exact answer to this. The test fails without a sudo.

Reply:
@szha @KellenSunderland  does this fall under the same sudo-discussion as https://github.com/apache/incubator-mxnet/pull/11317 ?

@aaronmarkham is it okay to change the installation instructions to force sudo?

Reply:
I can say that I lean towards not having `sudo`, but telling people they might need it. If we know the specific conditions, like only Mac users, or x state, then we should say so.
When you use `sudo`, it can cause downstream context problems which can be baffling and hard to diagnose, so I actually prefer using `pip install package --user` and keeping the installation context as narrow as possible. Would love to have someone that is an expert on this chime in a provide best practices. 

Also, isn't `nose` mostly for testing? Why tell the user they need to use `sudo` when they actually don't. They only need it if they want to run tests. Having them do something that is an edge case for CI or testing is the wrong approach.

Coming full circle, why are we doing this? The concept of automatically testing the instructions is good, but you shouldn't modify the instructions for the sake the test. 

Why don't you parse the install instructions, and inject `sudo` yourself as part of the script?

Reply:
Ye the nose-discussion is a different one, just wanted it for reference. But I agree - it should not be necessary to run pip as sudo since we don't expect our users to do it either.

Fully agree.